src/components/PriceBar.js

```diff
@@ -9,7 +9,7 @@ const Wrapper = styled.div`
   padding: 20px;
 `;
 
-export default function RangeBar() {
+export default function PriceBar() {
   const [selectedValue, setSelectedValue] = useState("");
```

Comment:
This state was replaced by the states filters and setFilters i moved down. Need here the connection between price bar and the state of the filters.

Reply:
With "the states" you probably mean the state `filters` and its corresponding function to change the states value `setFilters`